database/2share.sql

```diff
@@ -0,0 +1,21 @@
+CREATE DATABASE toshare;
+
+USE toshare;
+
+DROP TABLE if exists family;
+DROP TABLE if exists calendar;
+
+CREATE TABLE family 
+(
+id INT PRIMARY KEY NOT NULL AUTO_INCREMENT,
+name VARCHAR (100)
+);
+
+CREATE TABLE calendar
```

Comment:
a calendar doesn't have start/end dates: what will be stored in this table?

Reply:
we changed name table for 'event' instead of 'calendar'.
i pushed to my branch 'database'.